### PR TITLE
combined configuration builder: udpates for MathJax v2.6

### PR DESCRIPTION
**Pull request by pkra**
* update existing configurations: add AssistiveMML extension
* initial baseline for new configurations

### CODE REVIEW

**Comment by pkra** [comment]
Lots of blank files right now. 

Does `AM_CHTML(-full).lis` look ok?

**Comment by pkra** [comment]
Oh, I need to change CommonHTML to FastHTML everywhere...

**Comment by pkra** [comment]
And FHTML Preview.

**Comment by dpvc** [comment]
Except we may be changing FastHTML to PreviewHTML.

**Comment by pkra** [comment]
Yes! Sorry.

**Comment by pkra** [comment]
Gah. Would that turn FHTML into PHTML?

**Comment by pkra** [comment]
Let's make a decision quickly ;-)

**Comment by dpvc** [comment]
> Would that turn FHTML into PHTML?

Yes.

**Comment by pkra** [comment]
How about a more neutral `fast-preview.js` instead?

**Comment by dpvc** [comment]
OK.  With the output jax renamed to PreviewHTML, right?

**Comment by pkra** [comment]
Yes, let's do that.

**Comment by dpvc** [comment]
Or should the extension be `preview-html.js`?

**Comment by pkra** [comment]
Either is fine with me.

**Comment by dpvc** [comment]
On reflection `fast-preview.js` is better.

**Comment by pkra** [comment]
Updated the PR. Ready for review.

File: combiner/lists/AM_SVG-full.lis [hunk 1]
```diff
@@ -0,0 +1,14 @@
+jax/input/AsciiMath/config.js
+jax/output/SVG/config.js
+jax/output/FastHTML/config.js
+extensions/asciimath2jax.js
+extensions/MathEvents.js
+extensions/MathZoom.js
+extensions/MathMenu.js
+jax/element/mml/jax.js
+extensions/toMathML.js
```
Comment (dpvc) [inline combiner/lists/AM_SVG-full.lis]: The `-full` version should include `jax/input/AsciiMath/jax.js` in addition to the `config.js` file.

In fact, even the versions without `-full` have included the input jax in the past.  They probably shouldn't have, but they all do, so I don't know if you want to do that for `AM_SVG` below or not (it is included in `AM_CHTML` above, so I would think so).

File: combiner/lists/MML_SVG-full.lis [hunk 2]
```diff
@@ -0,0 +1,14 @@
+jax/input/MathML/config.js
+jax/output/SVG/config.js
+jax/output/FastHTML/config.js
+extensions/mml2jax.js
+extensions/MathEvents.js
+extensions/MathZoom.js
+extensions/MathMenu.js
+jax/element/mml/jax.js
+extensions/toMathML.js
```
Comment (dpvc) [inline combiner/lists/MML_SVG-full.lis]: Again, the `jax/input/MathML/jax.js` should be included here, and probably in `MML_SVG` below.

File: combiner/lists/AM_SVG-full.lis (hunk 1 above)
Comment (pkra) [inline combiner/lists/AM_SVG-full.lis]: Copy that. Do you want me to do it?

File: combiner/lists/TeX-MML-AM_CHTM-fullL.lis [hunk 3]
```diff
@@ -0,0 +1,22 @@
+jax/input/TeX/config.js
```
Comment (dpvc) [inline combiner/lists/TeX-MML-AM_CHTM-fullL.lis]: The name of this file has an extra `L` after `-full` (`TeX-MML-AM_CHTM-fullL.lis`).  The file needs to be renamed.

File: combiner/lists/AM_SVG-full.lis (hunk 1 above)
Comment (dpvc) [inline combiner/lists/AM_SVG-full.lis]: Sure, it's your PR, I'm happy for you to do it, but I can do it if you don't want to.

Comment (pkra) [inline combiner/lists/AM_SVG-full.lis]: I'll get on it.

**Comment by dpvc** [comment]
You should also change `FastHTML` to `PreviewHTML`, since that is the name we (eventually) agreed to.

File: combiner/lists/TeX-MML-AM_CHTM-fullL.lis (hunk 3 above)
Comment (pkra) [inline combiner/lists/TeX-MML-AM_CHTM-fullL.lis]: Yup, the L needs to move to complete CHTML

File: combiner/lists/MML_SVG-full.lis (hunk 2 above)
Comment (pkra) [inline combiner/lists/MML_SVG-full.lis]: Copy that.

**Comment by pkra** [comment]
Ready for review.

**Comment by pkra** [comment]
Eek. Had to amend that commit real quick (dropped the renamed file...)

**Comment by dpvc** [comment]
Note:  these new configuration files will need to be documented.

**Pull request merged**